src/components/ModalDelete.tsx

```diff
@@ -0,0 +1,32 @@
+import { FunctionComponent } from 'react'
+
+import Button from 'react-bootstrap/Button'
+import Modal from 'react-bootstrap/Modal'
+
+interface ModalProps {
+  handleClose: any
+  show: boolean
+  onDelete: any
+}
+
+const ModalDelete: FunctionComponent<ModalProps> = ({ handleClose, show, onDelete }) => {
+  return (
+    <>
+      <Modal show={show} onHide={handleClose}>
+        <Modal.Header>
+          <Modal.Title>Confirmation</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>Etes vous sur de vouloir supprimer ce patient ?</Modal.Body>
+        <Modal.Footer>
+          <Button variant='danger' onClick={() => handleClose(false)}>
+            Retour
+          </Button>
+          <Button variant='primary' onClick={onDelete}>
+            Supprimer
+          </Button>
```

Comment:
I think the color code should be almost the opposite. The delete button should be red and the cancel should not be red

Reply:
fix